### DISCUSSION TITLE
Can the data set for CoqGym with PyCoq

### DISCUSSION THREAD

**Discussion by brando90** [princeton-vl/CoqGym#70] [Q&A] [answered]
Can CoqGym and it's data set be used with PyCoq?

https://github.com/ejgallego/pycoq 

https://github.com/ejgallego/pycoq/issues/20

**Comment by yangky11** [comment, marked as answer]
PyCoq is pretty new, and I'm not familiar with it. It looks like a Python wrapper of SerAPI. CoqGym has its own customized wrapper of SerAPI ([serapi.py](https://github.com/princeton-vl/CoqGym/blob/master/serapi.py)), though PyCoq should be more comprehensive and easy to use. 

I think it's possible and probably easier if the codebase of CoqGym uses PyCoq. But I don't think you can plug in PyCoq in the current codebase trivially.
